### PR TITLE
chore(deps): update browserslist db

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4916,9 +4916,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001109, caniuse-lite@^1.0.30001125, caniuse-lite@^1.0.30001208:
-  version "1.0.30001210"
-  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001210.tgz"
-  integrity sha512-avmGf0Jo00I8vB0I89J4Pba48kddasErV7slu7wrkyM5uY9gE5P+B+V3hjABv8Hp4YNG2nBqIUFUXlnqNteXEA==
+  version "1.0.30001283"
+  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001283.tgz"
+  integrity sha512-9RoKo841j1GQFSJz/nCXOj0sD7tHBtlowjYlrqIUS812x9/emfBLBt6IyMz1zIaYc/eRL8Cs6HPUVi2Hzq4sIg==
```
Comment (lukaw3d) [inline yarn.lock:4921]: Still transpiles awaits and generators in production :sweat: 
https://github.com/oasisprotocol/oasis-wallet-web/blob/6639db1f1f66610295900f6aad97d5b279a15445/package.json#L37-L42

File: yarn.lock [hunk 2]
```diff
@@ -4916,9 +4916,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001109, caniuse-lite@^1.0.30001125, caniuse-lite@^1.0.30001208:
-  version "1.0.30001210"
```
Comment (pro-wh) [inline yarn.lock:4919]: :information_source: "7 months ago" according to npmjs.com

File: yarn.lock [hunk 3]
```diff
@@ -4916,9 +4916,9 @@ caniuse-api@^3.0.0:
...
-  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001210.tgz"
-  integrity sha512-avmGf0Jo00I8vB0I89J4Pba48kddasErV7slu7wrkyM5uY9gE5P+B+V3hjABv8Hp4YNG2nBqIUFUXlnqNteXEA==
+  version "1.0.30001283"
```
Comment (pro-wh) [inline yarn.lock:4919]: :information_source: "9 days ago" according to npmjs.com

a newer version `1.0.30001284` is available from "13 hours ago"

**Review by pro-wh — APPROVED** [review]
huh, so we don't have any process in place for updating this. nothing against doing it at all, so go ahead

**Comment by lukaw3d** [comment]
> so we don't have any process in place for updating this

Yeah, renovate doesn't seem to catch it. It isn't very important, might just contain some unneeded polyfills as random internet users upgrade their browsers.

caniuse-lite does spam developer's terminal on every `yarn start` :shrug: 
```
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
```

**Pull request merged**